server/Game.js

```diff
@@ -156,19 +173,17 @@ export default class {
 
   sendWordCandidates() {
     const player = this.nextPlayer();
+
```

Comment:
nit: remove this extra newline?

Reply:
On this again, the reason was to make it more readable :-) I like that you extracted `shouldClearWordsPlayed`, I'd suggest grouping the code like this:
```js
const player = this.nextPlayer();
if (!player) return;

this.currentPlayer = player.id;

const shouldClearWordsPlayed =
    this.words.size > words.length / 2 ||
    words.length - this.words.size < MAX_CANDIDATES;
if (shouldClearWordsPlayed) this.words.clear();

```